app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +17,8 @@
         <activity
             android:name=".MainActivity"
             android:exported="true"
-            android:screenOrientation="portrait">
+            android:screenOrientation="portrait"
+            android:theme="@style/Theme.MySplash">
```

Comment:
@amit-metri  rename "Theme.MySplash" to "Theme.BC_VAX.Splash

Reply:
done